ebu_tt_live/bindings/converters/ebutt1_ebutt3.py

```diff
@@ -1,18 +1,260 @@
+from ebu_tt_live.bindings import tt, tt_type, tt1_tt_type, tt1_body_type, \
+    body_type, div_type, tt1_head_type, tt1_layout_type, p_type, span_type,  \
+    br_type, head_type, style_type, styling, layout, \
+    region_type, ebuttdt, StyledElementMixin
+from ebu_tt_live.bindings._ebuttm import headMetadata_type, documentMetadata, \
+    metadataBase_type
+from ebu_tt_live.documents import EBUTT3Document
+from pyxb.binding.basis import NonElementContent, ElementContent
+import copy
 import logging
+from pyxb import ValidationError
 
 log = logging.getLogger(__name__)
 
 
 class EBUTT1EBUTT3Converter(object):
 
-    def __init__(self):
+    _semantic_dataset = None
+    _sequenceIdentifier = None
+
+    def __init__(self, sequence_id):
```

Comment:
By adding this extra `sequence_id` parameter, the converter's usage in `converters.py` won't work. Can you set a default value here?
Alternatively set a value in `converters.py`

Reply:
Yes, a value should be set in `converters.py` but I'm now wondering why that file even exists!

Reply:
I was thinking we'd use those helper functions in Distiller - they seem pretty handy to me.

Reply:
OK we haven't discussed exactly how we're going to instantiate the classes and run them in Distiller yet, so we should do that. My thinking was that we would be in closer control within the Distiller component of exactly how we do the conversion. For example wiring in the Denester etc.